WPMedia/fusion-news-theme-blocks
# Tmedia 15 float left right article body

## Description
On Desktop, float images left or right based on position 

## Jira Ticket
- [TMEDIA-15](https://arcpublishing.atlassian.net/secure/RapidBoard.jspa?rapidView=875&modal=detail&selectedIssue=TMEDIA-15&assignee=5e387d6a1b1d910e5dfd7a9b)

## Acceptance Criteria
When photos within the article body has a right or left float applied, the photo should render on the right or left side of desktop/tablet view, like the example screenshot shows above. 

Text should wrap around the photo with appropriate 24px margin spacing

Center and none floats remain as-is (full width)

Mobile view remains as-is (no float or text wrapping on mobile)

Limiting height for portrait orientation photos will be addressed in TMEDIA-321: Limit height of photo embeds within article bodyPO GROOMING and should be released together

Tests and documentation are updated as necessary

## Test Steps
1. Checkout this branch `git checkout TMEDIA-15-float-left-right-article-body`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/article-body-block`
3. Go to and see text move around the old-fashion image  http://localhost/pf/entertainment/2021/06/28/jenaes-global-kitchen-sink-article/?_website=the-gazette


## Effect Of Changes
### Before

<img width="1002" alt="Screen Shot 2021-07-16 at 14 29 22" src="https://user-images.githubusercontent.com/5950956/125999274-7a2ac380-4861-40a7-afcf-b3112e7ff582.png">

### After

<img width="785" alt="Screen Shot 2021-07-16 at 14 27 51" src="https://user-images.githubusercontent.com/5950956/125999134-6219ca50-d863-4e06-a286-bc557f681f88.png">

## Dependencies or Side Effects

## Deferred
- This is only on desktop

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
